docs/source/serialization.rst

```diff
@@ -0,0 +1,433 @@
+.. 
+    Copyright 2020 The HuggingFace Team. All rights reserved.
```

Comment:
This file shouldn't be included (must be a Git problem with the switch from RST to MDX). Could you please delete it from the PR?

Reply:
As you say

Reply:
Hmm I still see the file in the PR - is it possible you only reverted the diff? If a normal `rm docs/source/serialization.rst` + git commit doesn't fix it, perhaps running 

```
git rm docs/source/serialization.rst
```

does?

Reply:
Evsen should be fine now